README.md

```diff
@@ -117,6 +117,7 @@ If you're having issues with the packaged executable or using this tool in gener
 | Russian (Русский)                           | ru    | Oleg                                          | UI                                   |
 | Spanish (Español)                           | sp    | [enriiquee](https://github.com/enriiquee)     | UI                                   |
 | Spanish Latam (Español Latam)               | sp_la | [Matyrela](https://github.com/Matyrela)       | UI                                   |
+| Serbian									  | sr    | [rina](https://github.com/sweatshirts)        | UI                                   |
```

Comment:
Looks like you might be mixing tabs and spaces here. Just need the alignment fixed - content is good.

Reply:
formatting has been fixed!